rotkehlchen/chain/ethereum/modules/yearn/vaults.py

```diff
@@ -1,340 +1,10 @@
-                token=A_YV1_RENBT_CRV.resolve_to_evm_token(),
-            ),
-            'yvusdn3CRV': YearnVault(
-                name='curve.fi/usdn Vault',
-                contract=self.ethereum.contracts.contract(string_to_evm_address('0xFe39Ce91437C76178665D64d7a2694B0f6f17fE3')),
-                token=A_YV1_USDN_CRV.resolve_to_evm_token(),
-            ),
-            'yvust3CRV': YearnVault(
-                name='curve.fi/ust Vault',
-                contract=self.ethereum.contracts.contract(string_to_evm_address('0xF6C9E9AF314982A4b38366f4AbfAa00595C5A6fC')),
-                token=A_YV1_UST_CRV.resolve_to_evm_token(),
-            ),
-            'yvbBTC/sbtcCRV': YearnVault(
-                name='curve.fi/bbtc Vault',
-                contract=self.ethereum.contracts.contract(string_to_evm_address('0xA8B1Cb4ed612ee179BDeA16CCa6Ba596321AE52D')),
-                token=A_YV1_BBTC_CRV.resolve_to_evm_token(),
-            ),
-            'yvtbtc/sbtcCrv': YearnVault(
-                name='curve.fi/tbtc Vault',
-                contract=self.ethereum.contracts.contract(string_to_evm_address('0x07FB4756f67bD46B748b16119E802F1f880fb2CC')),
-                token=A_YV1_TBTC_CRV.resolve_to_evm_token(),
-            ),
-            'yvoBTC/sbtcCRV': YearnVault(
-                name='curve.fi/obtc Vault',
-                contract=self.ethereum.contracts.contract(string_to_evm_address('0x7F83935EcFe4729c4Ea592Ab2bC1A32588409797')),
-                token=A_YV1_OBTC_CRV.resolve_to_evm_token(),
-            ),
-            'yvhCRV': YearnVault(
-                name='curve.fi/hbtc Vault',
-                contract=self.ethereum.contracts.contract(string_to_evm_address('0x46AFc2dfBd1ea0c0760CAD8262A5838e803A37e5')),
-                token=A_YV1_HBTC_CRV.resolve_to_evm_token(),
-            ),
-            'yvcrvPlain3andSUSD': YearnVault(
-                name='curve.fi/susd Vault',
-                contract=self.ethereum.contracts.contract(string_to_evm_address('0x5533ed0a3b83F70c3c4a1f69Ef5546D3D4713E44')),
-                token=A_YV1_SUSD_CRV.resolve_to_evm_token(),
-            ),
-            'yvhusd3CRV': YearnVault(
-                name='curve.fi/husd Vault',
-                contract=self.ethereum.contracts.contract(string_to_evm_address('0x39546945695DCb1c037C836925B355262f551f55')),
-                token=A_YV1_HUSD_CRV.resolve_to_evm_token(),
-            ),
-            'yvdusd3CRV': YearnVault(
-                name='curve.fi/dusd Vault',
-                contract=self.ethereum.contracts.contract(string_to_evm_address('0x8e6741b456a074F0Bc45B8b82A755d4aF7E965dF')),
-                token=A_YV1_DUSD_3CRV.resolve_to_evm_token(),
-            ),
-            'yva3CRV': YearnVault(
-                name='curve.fi/aave Vault',
-                contract=self.ethereum.contracts.contract(string_to_evm_address('0x03403154afc09Ce8e44C3B185C82C6aD5f86b9ab')),
-                token=A_YV1_A3CRV.resolve_to_evm_token(),
-            ),
```

Comment:
Some of those constant assets were used only here so they can be removed too

Reply:
I plan to do a few rounds of dead code linting after the PR to find stuff like this